### ISSUE TITLE
New line added on windows but works fine for linux for 5.0.3

### ISSUE THREAD

**Issue by AlexandreYang** [timothycrosley/isort#1278]
Hi, first thanks for the awesome work on isort,

It seems there is an issue with isort `5.0.3` on Windows. 
The example below is failing on Windows but works on Linux.

`isort_test_file.py:`
```
try:
    import datadog_agent

    from ..log import CheckLoggingAdapter, init_logging

    init_logging()
except ImportError:
    pass
```

Command used:
```
isort --check-only --diff isort_test_file.py
```

Output on windows:

```
2020-07-06T10:32:38.8847283Z ERROR: D:\a\1\s\isort_test_file.py Imports are incorrectly sorted and/or formatted.
2020-07-06T10:32:38.8848516Z --- D:\a\1\s\isort_test_file.py:before	2020-07-06 10:30:34.221084
2020-07-06T10:32:38.8850018Z +++ D:\a\1\s\isort_test_file.py:after	2020-07-06 10:32:38.883964
2020-07-06T10:32:38.8850842Z @@ -1,6 +1,7 @@
2020-07-06T10:32:38.8851503Z  try:
2020-07-06T10:32:38.8852062Z 
2020-07-06T10:32:38.8852487Z      import datadog_agent
2020-07-06T10:32:38.8852759Z 
2020-07-06T10:32:38.8853257Z  
2020-07-06T10:32:38.8853464Z 
2020-07-06T10:32:38.8854079Z +
2020-07-06T10:32:38.8854581Z      from ..log import CheckLoggingAdapter, init_logging
2020-07-06T10:32:38.8854849Z 
2020-07-06T10:32:38.8855776Z  
2020-07-06T10:32:38.8855957Z 
2020-07-06T10:32:38.8856123Z      init_logging()
```

[Azure Pipeline CI Build link](https://dev.azure.com/datadoghq/integrations-core/_build/results?buildId=19176&view=logs&j=9720347a-34cb-5824-38aa-606c39f81a9d&t=75e8b099-9859-57c7-277c-194608322977)
[Example Code](https://github.com/DataDog/integrations-core/pull/7063)

Python version: 3.8.3
Windows image: windows-2019

**Comment by AlexandreYang** [comment]
Possibly same as (or related) https://github.com/timothycrosley/isort/issues/1277

**Comment by timothycrosley** [comment]
Thanks for reporting this bug! You were correct that it was highly related to #1277. A fix was just released in 5.0.8 with an additional regression test added.

**Comment by AlexandreYang** [comment]
Awesome, thanks!
